docs/acme-divergences.md

```diff
@@ -59,6 +59,13 @@ Boulder does not implement applications, instead it implements the `new-cert` fl
 
 Boulder does not implement the `reason` field for the `revoke-cert` endpoint, `unspecified` (0) from [RFC3280 Section 5.3.1](https://tools.ietf.org/html/rfc3280#section-5.3.1) is used for all requests.
 
+## [Section 6.6.](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-6.6)
+
+Boulder considers the following keys authorized to revoke a certificate:
+
+1. The account key that initially created the certificate being revoked
+2. The public key in the certificate being revoked
```

Comment:
I would specifically add another sentence: The ACME protocol allows for revocation of a certificate by an account key that is authorized for all DNS names in a certificate, even when that account did not create the certificate. However, that is not yet supported by Boulder.